src/app/personside/dialogpanel/sendMelding/SendNyMelding.tsx

```diff
@@ -1,119 +1,172 @@
 import * as React from 'react';
-import { FormEvent, useEffect, useState } from 'react';
-import { RadioPanelGruppe, Textarea } from 'nav-frontend-skjema';
-import { Meldingstype } from '../../../../models/meldinger/meldinger';
+import { FormEvent, useState } from 'react';
+import { KommunikasjonsKanal, Meldingstype } from '../../../../models/meldinger/meldinger';
 import { Kodeverk } from '../../../../models/kodeverk';
 import { UnmountClosed } from 'react-collapse';
 import KnappBase from 'nav-frontend-knapper';
 import styled from 'styled-components';
 import Temavelger from '../component/Temavelger';
-import { SkjemaelementFeil } from 'nav-frontend-skjema/lib/skjemaelement-feilmelding';
 import KnappMedBekreftPopup from '../../../../components/KnappMedBekreftPopup';
-import { useDispatch } from 'react-redux';
-import { sendMeldingActionCreator } from '../../../../redux/restReducers/sendMelding';
+import { useDispatch, useSelector } from 'react-redux';
+import { AppState } from '../../../../redux/reducers';
+import { JournalforingsSak } from '../../infotabs/meldinger/traadvisning/verktoylinje/journalforing/JournalforingPanel';
+import DialogpanelVelgSak from './DialogpanelVelgSak';
+import { isLoadedPerson } from '../../../../redux/restReducers/personinformasjon';
+import { capitalizeName } from '../../../../utils/stringFormatting';
+import AlertStripeInfo from 'nav-frontend-alertstriper/lib/info-alertstripe';
+import { getSaksbehandlerEnhet } from '../../../../utils/loggInfo/saksbehandlersEnhetInfo';
+import { NyMeldingValidator } from './validatorer';
+import TekstFelt from './TekstFelt';
+import VelgDialogType from './VelgDialogType';
+import { useRestResource } from '../../../../utils/customHooks';
+import { Undertittel } from 'nav-frontend-typografi';
+import Oppgaveliste from './Oppgaveliste';
+import { isPosting } from '../../../../rest/utils/postResource';
+
+export enum OppgavelisteValg {
+    MinListe = 'MinListe',
+    EnhetensListe = 'Enhetensliste'
+}
+
+export enum SendNyMeldingDialogTyper {
```

Comment:
her er kun 3 meldingstyper definert (2 notat og ein inngåande spørmål) skulle utgående spørsmål, utgående svar og inngående svar også vere definert her?

Reply:
Så vidt jeg ser fra den gamle frontenden tror jeg bare disse tre typene kan brukes for ny melding? Utgående svar blir vel i fortsett dialog (ny komponent), og inngående svar kan man vel neppe sende fra modia?

Reply:
Aha, ok, så da byttes SPOSMAL_SKRIFTLIG til SPORSMAL_MODIA_UTGAAENDE